### PR TITLE
raw ops: support for double and float; add translation CID->ADDR

### PR DESCRIPTION
**Pull request by rucub100**

### CODE REVIEW

File: src/main/java/de/hhu/bsinfo/dxmem/operations/Pinning.java [hunk 1]
```diff
@@ -28,6 +28,7 @@
  * Pin a chunk to allow direct access to heap data using the address (and for RDMA)
  *
  * @author Stefan Nothaas, stefan.nothaas@hhu.de, 31.08.2018
+ * @author Ruslan Curbanov, ruslan.curbanov@uni-duesseldorf.de, 18. März 2019
```
Comment (krakowski) [inline src/main/java/de/hhu/bsinfo/dxmem/operations/Pinning.java]:
```suggestion
 * @author Ruslan Curbanov, ruslan.curbanov@uni-duesseldorf.de, 18.03.2019
```

File: src/main/java/de/hhu/bsinfo/dxmem/operations/Pinning.java [hunk 2]
```diff
@@ -106,6 +107,18 @@ public PinnedMemory pin(final long p_cid, final int p_acquireLockTimeoutMs) {
         return new PinnedMemory(tableEntry.getAddress());
     }
 
+    /**
+     * Translates a pinned chunk ID into the raw address.
+     *
+     * @param p_cid The chunk ID to translate
+     * @return The raw address of the chunk im memory
```
Comment (krakowski) [inline src/main/java/de/hhu/bsinfo/dxmem/operations/Pinning.java]:
```suggestion
     * @return The raw address of the chunk in memory
```

File: src/main/java/de/hhu/bsinfo/dxmem/operations/Pinning.java (hunk 1 above)
Comment (rucub100) [reply src/main/java/de/hhu/bsinfo/dxmem/operations/Pinning.java]: done

File: src/main/java/de/hhu/bsinfo/dxmem/operations/Pinning.java (hunk 2 above)
Comment (rucub100) [reply src/main/java/de/hhu/bsinfo/dxmem/operations/Pinning.java]: done

**Pull request merged**